components/selection/selection-observer-mixin.js

```diff
@@ -7,7 +7,12 @@ export const SelectionObserverMixin = superclass => class extends superclass {
 			/**
 			 * The selection info (set by the selection component).
 			 */
-			selectionInfo: { type: Object }
+			selectionInfo: { type: Object },
+			/**
+			 * Id of the collection this component wants to subscribe to (if not located with that collection)
+			 */
+			subscribedTo: { type: String, reflect: true, attribute: 'subscribed-to' },
+			_provider: { type: Object, attribute: false }
```

Comment:
Added this for the case where the collection later registers the components, so that this will cause a re-render of the `selection-select-all` and the `selection-summary` components.  They hide themselves if the collection has the `selection-single` attribute.